nursery/get-current-process-command-line.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: get current process command line
```

Comment:
add `on Linux`?

Reply:
i figured we should split out that naming once we have multiple implementations. thoughts?

Reply:
yeah, that works